dev-util/gammaray/gammaray-9999.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+QT_MINIMAL="5.9"
+
+DESCRIPTION="High-level runtime introspection tool for Qt applications"
+HOMEPAGE="https://github.com/KDAB/GammaRay \
+		https://www.kdab.com/?s=gammaray"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
+SLOT=0
+IUSE="3d auto-detect backtrace backtrace-symbol bluetooth +client \
+	concurrent designer -doc feedback +gpl-only-features \
+	position printsupport scxml svg test qml unwind \
+	wayland webengine"
+
+RDEPEND="
+	$(add_qt_dep qtcore)
+	$(add_qt_dep qtgui)
+	$(add_qt_dep qtnetwork)
+	$(add_qt_dep qtwidgets)
+	$(add_qt_dep qtxml)
+	$(add_frameworks_dep kitemmodels)
+"
+
+DEPEND="${RDEPEND}
+	3d? ( $(add_qt_dep qt3d) )
+	bluetooth? ( $(add_qt_dep qtbluetooth) )
+	concurrent? ( $(add_qt_dep qtconcurrent) )
+	designer? ( $(add_qt_dep designer) )
+	position? ( $(add_qt_dep qtpositioning) )
+	printsupport? ( $(add_qt_dep qtprintsupport) )
+	qml? ( $(add_qt_dep qtdeclarative) )
+	scxml? ( $(add_qt_dep qtscxml) )
+	svg? ( $(add_qt_dep qtsvg) )
+	test? ( $(add_qt_dep qttest) )
+	webengine? ( $(add_qt_dep qtwebengine) )
```

Comment:
further down in src_configure you disable QtWebengineWidgets, so this also needs to use-depend on `qtwebengine[widgets]` here.